Gopkg.lock

```diff
@@ -123,6 +123,12 @@
 
 [[projects]]
   branch = "master"
+  name = "github.com/golang/sync"
+  packages = ["errgroup"]
+  revision = "f52d1811a62927559de87708c8913c1650ce4f26"
```

Comment:
hmm I think we can uncommit this stuff, I just ran `dep ensure` and I'm at the latest of each dep